src/renderer/survey/js/survey.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * The survey component in the renderer view
+ *
+ * A wrapper for the survey popup and the channel used for IPC between the main, the
+ * renderer process and the webview housing the survey web page.
+ * Copyright 2017 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+/* global fluid */
+
+"use strict";
+/**
+ * A wrapper component for the `gpii.survey.channel` (which handles the
+ * communication between the `BrowserWindow` and the main electron process)
+ * and the `gpii.survey.popup` (which wraps and serves as an intermediary
+ * for the webview).
+ */
+(function (fluid) {
```

Comment:
File closure seems unnecessary

Reply:
As agreed on a call with Antranig on 13 Feb, this comment will not be addressed now. The reasons for this are that there will be huge diffs (because indentation will need to be corrected) as closure is used everywhere in the `renderer` part of the project. Also, there are a few branches which have stemmed either from this one or from master and there will be conflicts when merging them due to the indentation changes.